h/static/styles/tags-input.scss

```diff
@@ -101,3 +101,47 @@ tags-input {
   }
 }
 
+tags-input .autocomplete {
```

Comment:
Okay, I've possibly done that. Somebody with stronger css skills can look into that.